src/Public/js/nipsweb.channelconfig.js

```diff
@@ -0,0 +1,130 @@
+var ChannelConfigurator = function(player) {
+
+    var getOrUpdateStoredDeviceMappings = function(devices) {
+        var map = [];
+        var unmapped = [];
+        var mapped;
+
+        var storedDevices = [];
+
+        try {
+            if (localStorage && localStorage.hasOwnProperty('nipsWebKnownDevices')) {
+                storedDevices = localStorage.nipsWebKnownDevices;
+            }
+        } catch (e) {console.error('Local Storage is being mean.', e);}
+
+        for (var i = 0; i < devices.length; i++) {

```

Comment:
Sorry, I'm used to having to support oldIE etc, which doesn't have such fancy things. Might even get some ES2015 in here instead :P